.bazelrc

```diff
@@ -125,6 +125,10 @@ build:remote --bes_results_url="https://source.cloud.google.com/results/invocati
  # Set remote caching settings
 build:remote --remote_accept_cached=true
 
+# Force remote exeuctions to consider the entire run as linux
```

Comment:
nit: this is copied from angular/angular and is required for OSX cross-platform RBE. the comment could use an explanation of why it is there: "required for OSX cross-platform RBE".

Reply:
Added